tests/validators/test_uuid.py

```diff
@@ -0,0 +1,164 @@
+import copy
+import re
+from uuid import UUID
+
+import pytest
+
+from pydantic_core import SchemaValidator, ValidationError
+
+from ..conftest import Err, PyAndJson
+
+
+@pytest.mark.parametrize(
+    'input_value,expected',
+    [
+        # Valid UUIDs
+        ('12345678-1234-1234-1234-567812345678', UUID('12345678-1234-1234-1234-567812345678')),
```

Comment:
Add a case with UUID input to confirm exact instances validate successfully?

Reply:
done 👍